### ISSUE TITLE
About the pretrain command on the COCO datset

### ISSUE THREAD

**Issue by myukzzz** [gpt4vision/OvSGTR#13]
Dear Author:

First of all, thank you very much for disclosing the code of your outstanding work, but I still want to repeat the pre-training process, can I just modify the config file corresponding to '-c' in the public training command? But this is still training on the vg dataset, which is not what you described in your article.

Please share the command to repeat the pre-training process. Thank you again for sharing the code of your outstanding work!

**Comment by JosephChenHub** [comment]
> Dear Author:
> 
> First of all, thank you very much for disclosing the code of your outstanding work, but I still want to repeat the pre-training process, can I just modify the config file corresponding to '-c' in the public training command? But this is still training on the vg dataset, which is not what you described in your article.
> 
> Please share the command to repeat the pre-training process. Thank you again for sharing the code of your outstanding work!

you can specify the dataset to coco captions instead of vg, and I have uploaded the parsed triplet files as 
```you can download [captions_train2017_triple.json](https://huggingface.co/JosephZ/OvSGTR/resolve/main/captions_train2017_triple.json) and [captions_val2017_triple.json](https://huggingface.co/JosephZ/OvSGTR/resolve/main/captions_val2017_triple.json) from huggingface``` in the [data.md](https://github.com/gpt4vision/OvSGTR/blob/master/datasets/data.md)

**Comment by myukzzz** [comment]
There are still missing files...

**Comment by JosephChenHub** [comment]
> coco_nouns.txt and coco_relations.txt in the annotations catalogue

well, please check the updated [data.md](https://github.com/gpt4vision/OvSGTR/blob/master/datasets/data.md)

**Comment by myukzzz** [comment]
Thanks for your reply!

However, there still seem to be problems with the pre-training process.

According to section 3.2 of the article and the published json file, it seems that the model does not need bbox supervision in the pre-training stage. However, in the code, it seems that bbox-related loss is added to both config and code flow for optimization

![d969265f246ac052578962791d34ced](https://github.com/user-attachments/assets/97d672c6-a3c4-4290-b080-78d1a890aadc)

**Comment by myukzzz** [comment]
![image](https://github.com/user-attachments/assets/ca40594b-0010-4281-94fa-1e52663f1fd1)
There does not appear to be bbox supervision in the published json file

**Comment by myukzzz** [comment]
![image](https://github.com/user-attachments/assets/7fa4219e-4d0f-424b-8dbc-08bd7aa9adb2)
There seems to be no bbox supervision in the published json file......
